app/Notifications/WelcomeEmailNotification.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Channels\SailthruChannel;
+use App\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class WelcomeEmailNotification extends Notification
+{
+    use Queueable;
+
+    /**
+     * @var string
+     */
+    public $sailThruTemplate = '';
+
+    /**
+     * @var array
+     */
+    public $sailThruParameters = [];
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return [
+            'mail',
+            SailthruChannel::class
+        ];
```

Comment:
Also, Cheki makes this more complicated as we configure the email driver per email and per domain.